eZ/Publish/Core/Persistence/Legacy/User/Gateway.php

```diff
@@ -1,92 +1,68 @@
 <?php
 
 /**
- * File containing the User Gateway class.
- *
  * @copyright Copyright (C) eZ Systems AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
+declare(strict_types=1);
```

Comment:
You missed `declare(strict_types=1); ` in  `\eZ\Publish\Core\Persistence\Legacy\Tests\User\{Gateway\DoctrineDatabaseTest,UserHandlerTest}` 😉

Reply:
> You missed `declare(strict_types=1); ` in `\eZ\Publish\Core\Persistence\Legacy\Tests\User\{Gateway\DoctrineDatabaseTest,UserHandlerTest}`

Added to `DoctrineDatabaseTest` via c830637. `UserHandlerTest` requires more changes for consistency's sake, but those changes overlap with User Role gateway refactoring I'm doing now. Refactoring of tests for common code will be done via [EZP-31246](https://jira.ez.no/browse/EZP-31246) (pending PR #2903). But first we need to finish production code changes to unblock it.